libnmstate/schemas/operational-state.yaml

```diff
@@ -456,3 +483,24 @@ definitions:
         type: array
         items:
           type: string
+  vlan-filter:
+    type: object
+    properties:
+      id:
+        $ref: "#/definitions/types/vlan-id"
+      id-range:
+        type: object
+        properties:
+          min:
+            $ref: "#/definitions/types/vlan-id"
+          max:
+            $ref: "#/definitions/types/vlan-id"
+        required:
+          - min
+          - max
```

Comment:
> Could you please impose this in the code (`validator` module)? The schema errors are not that easy to read when something fails.

You know my opinion about this - e.g. what's the point of using the schema then ?...

Also, I think the tests I've added show that it really is easy to read, since it features very specific strings about what has happened.

Never the less, I'll bend, since that's consistent with the reminder of the project.

Reply:
> You know my opinion about this - e.g. what's the point of using the schema then ?

This has been answered already. It is used to describe the API in a common format/language. It allows other tools to generate client code automatically and so on.

> Also, I think the tests I've added show that it really is easy to read, since it features very specific strings about what has happened.

If this is readable I am ok to keep it.
Could you please paster here an output from the shell when this happens?

Reply:
Will do.

Reply:
Ended up following your clue, and implementing such things in the validator module.